### PR TITLE
Multiple issue templates

### PR DESCRIPTION
**Pull request by karthik**
This will split the template for new submissions and a separate one for presubmission inquiries. Once this is merged, opening a new issue will present the user with 3 templates:
- meta issue
- presub
- software submission

### CODE REVIEW

**Comment by karthik** [comment]
@maelle If you want to see how this would look after merge, check out this [test repo](https://github.com/karthik/issues/issues)

**Comment by karthik** [comment]
Ok, I also added a 3rd template that directs people to onboarding meta for meta issues. Looks like this:

image

File: .github/ISSUE_TEMPLATE/submit-a-presubmission-inquiry.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+-   Paste the full DESCRIPTION file inside a code block below:
+
+```
+
+```
+
+
+## Scope 
+
+- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) this package falls under: (Please check an appropriate box below. If you are unsure, we suggest you make a pre-submission inquiry.):
```
Comment (noamross) [inline .github/ISSUE_TEMPLATE/submit-a-presubmission-inquiry.md]:
```suggestion
- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) this package falls under: (Please check an appropriate box below.:
```

File: .github/ISSUE_TEMPLATE/submit-a-presubmission-inquiry.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	- [ ] data extraction
+	- [ ] database access
+	- [ ] data munging
+	- [ ] data deposition
+	- [ ] reproducibility
+	- [ ] geospatial data
+	- [ ] text analysis
+	
+
+- Explain how the and why the package falls under these categories (briefly, 1-2 sentences):
```
Comment (noamross) [inline .github/ISSUE_TEMPLATE/submit-a-presubmission-inquiry.md]:
```suggestion
- Explain how the and why the package falls under these categories (briefly, 1-2 sentences).  Please note any areas you are unsure of:
```

File: .github/ISSUE_TEMPLATE/submit-a-presubmission-inquiry.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+
+-   Who is the target audience and what are scientific applications of this package?  
+
+-   Are there other R packages that accomplish the same thing? If so, how does yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?
```
Comment (noamross) [inline .github/ISSUE_TEMPLATE/submit-a-presubmission-inquiry.md]:
```suggestion
-   Are there other R packages that accomplish the same thing? If so, how does yours differ or meet [our criteria for best-in-category](https://github.com/ropensci/onboarding/blob/master/policies.md#overlap)?

-  Any other questions or issues we should be aware of?:
```

**Comment by noamross** [comment]
Made a few suggested edits.

**Comment by karthik** [comment]
Maëlle: It has been improved more now.

**Comment by maelle** [comment]
Cool! 

* I added links to the dev_guide

* I renamed the files because I think they appear in the alphabetical order from top to bottom?

No further comment from me.

**Pull request merged**